### ISSUE TITLE
Unable to logout/reset jotta-cli.

### ISSUE THREAD

**Issue by vegovs** [jotta/jotta-cli-issues#105]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
-------------------------------------------
jottad executable : 
jottad appdata    : /var/lib/jottad
jottad version    : 0.6.21799
jotta-cli version : 0.6.21799
-------------------------------------------

**Description of problem:**
Unable to logout/reset jotta-cli.

**Expected:**
Logout and reset jotta-cli.

**jotta-cli status (`jotta-cli status`):**
N\A, cli does not respond. 

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
N\A

**Traceback**
N\A

**Additional info:**
```
(base) user@computer:~$ jotta-cli logout
This will reset jotta-cli locally. Backup will stop. Continue?(y/n): y
```
Nothing happens after y is appended.

**Comment by Kimbsen** [comment]
Thanks for reporting this issue. We have identified the cause and will try to release a fix version early next week.

In the meantime you can delete the contents on /var/lib/jottad and then restart jottad which is basically the same thing as a logout.

**Comment by Kimbsen** [comment]
We have released version 0.6.23861 which fixes this
